osu.Game.Tests/Visual/Gameplay/TestSceneOverlayActivation.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Game.Configuration;
+using osu.Game.Overlays;
+using osu.Game.Rulesets;
+
+namespace osu.Game.Tests.Visual.Gameplay
+{
+    public class TestSceneOverlayActivation : OsuPlayerTestScene
+    {
+        private OverlayTestPlayer testPlayer;
+
+        [Resolved]
+        private OsuConfigManager mng { get; set; }
```

Comment:
let's use proper variable names please

```suggestion
        private OsuConfigManager config { get; set; }
```

(+ usages)